vm/src/stdlib/posix.rs

```diff
@@ -1975,4 +1979,21 @@ pub mod module {
         res.map_err(|err| err.into_pyexception(vm))?;
         Ok(vm.ctx.new_int(written as u64).into())
     }
+
+    #[cfg(target_os = "linux")]
+    #[pyfunction]
+    fn getrandom(size: usize, flags: OptionalArg<u32>, vm: &VirtualMachine) -> PyResult<Vec<u8>> {
```

Comment:
CPython takes an (essentially) `isize` here and fails with `EINVAL`. Not sure if we can currently set `errno` to what we want but we could take an `isize`, try to convert it to a `usize` and then bail on failure with an `OSError` (We currently raise `OverflowError`).

Reply:
Fixed in new commit.